prow/cluster/monitoring/mixins/prometheus/tide_alerts.libsonnet

```diff
@@ -48,6 +48,32 @@
               message: 'The Tide status update controllers loop period has averaged more than 2 minutes for the last 15 mins.',
             },
           },
+          {
+            alert: 'TidePoolErrorRateIndividual',
+            expr: |||
+              (max(sum(increase(tidepoolerrors[10m])) by (org, repo, branch)) or vector(0)) >= 3
+            |||,
+            'for': '5m',
+            labels: {
+              severity: 'warning',
+            },
+            annotations: {
+              message: 'At least one Tide pool encountered 3+ sync errors in a 10 minute window. If the TidePoolErrorRateMultiple alert has not fired this is likely an isolated configuration issue. See the <deck-url>/tide-history page.',
```

Comment:
See the <https://prow.k8s.io/tide-history|tide-history page>.

The [link format](https://api.slack.com/docs/message-formatting) in slack.

Reply:
Just found this cool tool ^_^

https://api.slack.com/docs/messages/builder?msg=%7B%22text%22%3A%22See%20the%20%3Chttps%3A%2F%2Fprow.k8s.io%2Ftide-history%7Ctide-history%20page%3E.%22%7D

Reply:
I added a commit to add links to all the Tide alerts. They link to the tide-history page and/or graphs at https://monitoring.prow.k8s.io.